Fix web install instructions (#8)
Vencord/Docs: src/content/docs/installing/index.mdx

To do so, simply run the same build command again.

Since this becomes tedious quickly, we also provide a command that watches for changes and rebuilds automatically whenever you make changes.

> Simply replace `build` with `watch`, for example `pnpm watch buildWeb`!

Reviewer: ```suggestion
Simply add the `--watch` flag, for example `pnpm buildWeb --watch`!
```
